packages/documentation/src/pages/platform/tools/feature-toggles.mdx

```diff
@@ -66,9 +66,9 @@ const FEATURE_FLAG_NAMES = Object.freeze({
 
 7. Submit a PR for each feature. Crosslinking the PRs in a comment will make it easier for the reviewers to check.
 
-8. Use the selector helper to build a selector for your feature toggle.
+8. Use the selector helper to build a [selector](https://daveceddia.com/redux-selectors/) for your feature toggle. See an example [here](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/gi/containers/ProfilePage.jsx#L118).
```

Comment:
We should probably pin the link to a specific commit so the line number doesn't get out-of-sync with what we're trying to point at.
```suggestion
8. Use the selector helper to build a [selector](https://daveceddia.com/redux-selectors/) for your feature toggle. See an example [here](https://github.com/department-of-veterans-affairs/vets-website/blob/f6178ac59b1d691e0f31fb36903b8ba54091b517/src/applications/gi/containers/ProfilePage.jsx#L118).
```